### PR TITLE
New features git_path, full path of git binary, and AJP connector.

### PR DESCRIPTION
**Pull request by f3rr**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -60,10 +60,13 @@
 
   # Reverse https proxy
   $proxy = {},
+  # Options for the AJP connector
+  $ajp   = {},
 
   # Git version
   $git_manage  = true,
   $git_version = 'installed',
+  $git_path = undef,
```
Comment (igalic) [inline manifests/init.pp:69]: this should be aligned with the above options.

File: manifests/init.pp [hunk 2]
```diff
@@ -99,6 +103,19 @@
     }
   }
 
+  if ! empty($ajp) {
```
Comment (igalic) [inline manifests/init.pp:106]: unless

**Comment by igalic** [comment]
general comments: all of those options require documentation. and tests.

also: I'm not a big fan of adding ajp. i understand that some (big) companies have policies of running JEE stuff over AJP, but tomcat has come a long way in the past 1901248472 years, and its HTTP implementation is more than adequate, so adding another onion layer on top of this module seems… unnecessary.

especially when it breaks failing tests.

**Comment by f3rr** [comment]
LOL. OK. wont be fixed from my side. Regards.

**Comment by igalic** [comment]
okay, thanks for the follow up

**Pull request closed without merging**